src/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -22,7 +22,7 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(CommonTestPath)\XunitTraitsDiscoverers\XunitTraitsDiscoverers.csproj">
+    <ProjectReference Include="..\..\Common\tests\XunitTraitsDiscoverers\XunitTraitsDiscoverers.csproj">
```

Comment:
This PR was started before the change to use a variable rather than relative directory path for project references.  We should be using CommonTestPath here, thanks for catching this, I'll update / validate that change.